src/app/components/search/SavedSearch.js

```diff
@@ -70,12 +71,45 @@ const SavedSearch = ({ routeParams }) => (
                       }
                     `}
                     deleteMessage={
-                      <FormattedMessage
-                        id="savedSearch.deleteMessage"
-                        defaultMessage="Are you sure? This list is shared among all users of {teamName}. After deleting it, no user will be able to access it."
-                        description="A message that appears when a user tries to delete a list, warning them that it will affect other users in their workspace."
-                        values={{ teamName: props.saved_search.team ? props.saved_search.team.name : '' }}
-                      />
+                      props.saved_search?.is_part_of_feeds ?
+                        <p>
```

Comment:
I would remove this `<p>` tag, it is not proper html to have the `<Alert..` component be a child of a `<p>` tag. You can replace it with the react `<>`